### DISCUSSION TITLE
Packagecloud distribution

### DISCUSSION THREAD

**Discussion by alexdelprete** [emqx/emqx#9727] [General]
I just noticed a v5.0.14 was released but if I update my server via apt, 5.0.12 is still the latest.

I used packagecloud repository, and indeed v5.0.12 is still the latest there.

Isn't it part of the release process? If it isn't, I'll probably decide to switch to docker, but I'd prefer to have EMQX on a dedicated instance.

Thanks for the support.

**Comment by id** [comment]
@alexdelprete I suppose packagecloud syncs from our official repo at packages.emqx.com (see https://www.emqx.io/downloads?os=Debian), but the latest version there is indeed v5.0.12. We are looking into it, thanks for the report.

**Reply by id** [reply]
@alexdelprete we triggered re-index of our repo, v5.0.14 is available there now. Hopefully packagecloud will pick it up soon.

**Comment by alexdelprete** [comment]
I can see 5.0.14 on packagecloud now, thanks for the quick response. Hope in future it will be kept in sync. :)

**Comment by alexdelprete** [comment]
@id Unfortunately the same happens for v5.0.15. It's a pity packagecloud sync is not integrated in the release cycle. :(

Hope it will get permanently fixed one day...

**Comment by id** [comment]
@alexdelprete v5.0.15 should be available now, and we are working to have this automated by the next release.

**Reply by alexdelprete** [reply]
thanks. I updated manually downloading the `.deb` file and upgraded with `dpkg -i`. Everytime I use this procedure, the service doesn't come up correctly, and I need to reboot the machine.

Hope this will work cleanly once packagecloud will be synced automatically and upgrading through `apt`.

**Comment by id** [comment]
https://github.com/emqx/emqx/pull/9841

**Reply by alexdelprete** [reply]
> #9841

was #9841 in production for 5.0.16 release? I don't see it in packagecloud yet.

**Reply by id** [reply]
@alexdelprete PR missed the release window, should work automatically starting from v5.0.17

**Reply by id** [reply]
v5.0.16 packages are uploaded now

**Reply by alexdelprete** [reply]
Thank you Ivan.
